### PR TITLE
icon button was completed

### PR DESCRIPTION
**Pull request by FeruzaKazakova**

### CODE REVIEW

File: src/components/UI/IconButtons.tsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { IconButton } from '@mui/material'
```
Comment (myktybek03) [inline src/components/UI/IconButtons.tsx:1]: it's okay

**Review by myktybek03 — APPROVED** [review]

File: src/components/UI/IconButtons.tsx [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+type Props = {
+  icon?: React.ReactNode
+}
+
+const IconButtons = ({ icon }: Props) => {
+  return <IconButton>{icon}</IconButton>
+}
```
Comment (Nurzada03) [inline src/components/UI/IconButtons.tsx:9]: every thing is Ok

**Review by Nurzada03 — APPROVED** [review]

**Review by Nurbolot00 — APPROVED** [review]

**Review by dilmuratsarybaev — APPROVED** [review]

File: src/components/UI/IconButtons.tsx [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import { IconButton } from '@mui/material'
+
+type Props = {
+  icon?: React.ReactNode
+}
+
+const IconButtons = ({ icon }: Props) => {
```
Comment (songhee24) [inline src/components/UI/IconButtons.tsx]: need to pass ...rest

**Review by songhee24 — CHANGES_REQUESTED** [review]

File: src/components/UI/IconButtons.tsx [hunk 4]
```diff
@@ -0,0 +1,11 @@
+import { IconButton } from '@mui/material'
+
+type Props = {
```
Comment (songhee24) [inline src/components/UI/IconButtons.tsx:3]: ладно сделаю merge но для ...restProps тоже нужна типизация

**Review by songhee24 — APPROVED** [review]

**Pull request merged**